guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -321,14 +321,23 @@ Let's start with the HTML we're working with:
 <button type="button">Pause</button>
 ```
 
-Next, let's add an event handler to the `Play` button:
+It might be tempting to try to handle this by managing the whole set of interactions at the component level. But there are three reasons we might want to tackle this using a modifier instead:
 
-```handlebars {data-filename="app/components/audio-player.hbs" data-diff="-3,+4"}
+1. Components don't have access to DOM elements directly. We'd have to render the page, push an element back up into the component class, and *then* wire it up. This would come with performance costs, because we would have to render twice before anything could work.
+2. We can't really use autotracking or 1-way data flow that way. If we wanted the caller of our component to be able to pass an argument in and have the component respond appropriately, we'd have to use something like `{{did-update}}`.
+3. None of the logic we built to manage playing and pausing would be reusable! It would be specific to just this component.
```

Comment:
```suggestion
1. Components don't have direct access to DOM elements. We have to render the page, push an element into the component class, then wire it up. These steps can cause a performance issue because we have to render the page twice in order for things to start working.
2. We can't use autotracking and one-way data flow. If we want the caller of a component to be able to pass an argument and have the component respond appropriately, we will have to use a solution like `{{did-update}}`.
3. The code for managing play and pause is not reusable. It is tied to the audio component.
```

Reply:
This is helpful! I went with some slightly different wording and structure here, while trying to get at these goals; see 414d25ee.